client/components/domains/domain-suggestion/index.jsx

```diff
@@ -25,6 +25,15 @@ class DomainSuggestion extends React.Component {
 		domain: PropTypes.string,
 	};
 
+	keyboardHandler = event => {
+		switch ( event.key ) {
+			case 'enter':
```

Comment:
These are not values from specs. Should be `"Enter"` (special key) and `" "` (just a space character, because they key is printable, not special). 

Lookup table: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values

Reply:
My bad! About to push the fix :)

Reply:
not a fan of the space character though :( 

`"Spacebar"` or `"Space"` would feel more appropriate to me but it is what it is.

Reply:
Agreed :/